src/shared/seccomp-util.c

```diff
@@ -197,40 +227,26 @@ const SystemCallFilterSet syscall_filter_sets[] = {
                 "_sysctl\0"
                 "afs_syscall\0"
                 "break\0"
-                "fattach\0"
-                "fdetach\0"
+                "create_module\0"
                 "ftime\0"
                 "get_kernel_syms\0"
-                "get_mempolicy\0"
```

Comment:
This is removed, but is not added anywhere. If this is on purpose, the commit message should mention it. Similarly for vm86.

According to tuxcall(2), afs_syscall,  break,  fattach,  fdetach,  ftime,  getmsg, getpmsg, gtty, isastream, lock, madvise1, mpx, prof, profil, putmsg, putpmsg, security, stty, tuxcall, ulimit, vserver are not implemented. It's pretty hard to verify this list, but for the few I checked, it is accurate. We should probably remove all of them from this list then.